### DISCUSSION TITLE
".update" is not returning a response body

### DISCUSSION THREAD

**Discussion by danielsumah** [prisma/prisma#15503] [Q&A] [answered]
I wrote this to update a transaction..

The transaction updates correctly in the database but my response body is empty. 

![image](https://user-images.githubusercontent.com/52764894/192097711-02c0dbd4-bb52-4bdc-82e8-013336b960c2.png)

empty response body..
![image](https://user-images.githubusercontent.com/52764894/192097767-f4b7cdd1-577f-416c-86e4-63bf2510cde2.png)

Any explanation for why this happens?

**Comment by janpio** [comment]
Put a `const result = await this.prisma...` in your code and then log that to see what Prisma itself returns. It very certainly is the modified object.

If that is the case, you know that this is a problem in your `updateTransaction` function or its usage on higher levels - not a Prisma problem.

**Comment by danielsumah** [comment]
@janpio  Thanks for your response..

Yes, I got the modified object. 

I moved up to the controller..

First, I was getting a promise but after awaiting and logging out th result'; i got the updated response body in the terminal but still an empty body on insomnia

![image](https://user-images.githubusercontent.com/52764894/192102236-0c2e09a5-105b-4adb-aa39-887299300fe0.png)

**Comment by danielsumah** [comment, marked as answer]
@janpio  this has been solved.. 

I had an httpstatus code of 204 (no reponse) as a decorator on my controller.. This was what stopped the response body from showing

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
